### PR TITLE
font-sourcecodepro: rename and update to 1.010

### PR DESCRIPTION
**Pull request by platerza**
- rename from `font-sourcecodepro` to `font-adobe-source-code-pro`, in the name of consistency with other font packages
- update to version 1.010
  - new version is distributed in two separate files (one for _roman_ and one for _italics_) instead of one; `distfiles` and `checksum` were updated accordingly
- update and clean up the template instructions
  - use `vinstall` instead of `install`

### CODE REVIEW

**Comment by platerza** [comment]
@maxice8 just committed that metapackage. Since this is only transitional, I left the template bare-bones. It seemed appropriate not to specify:

- `maintainer`
- `license`
- `homepage`

as these are all filled out in the real package.

However, if you feel otherwise I can add those.

**Comment by maxice8** [comment]
I mean create a subpackage of the new package that is build_style=meta and takes the name of the package being removed

**Comment by maxice8** [comment]
see `gnome-tweaks` template for an example with `gnome-tweak-tool`

**Comment by maxice8** [comment]
Looks fine, please squash all commits into one.

**Comment by platerza** [comment]
On my cloned repo I made `srcpkgs/font-sourcecodepro/` a symbolic link to `srcpkgs/font-adobe-source-code-pro` (to match the gnome example) but after pushing it seems like git sees it as the contents of the link and not a symlink itself. Not sure how to do that.

**Comment by maxice8** [comment]
On Fri, Aug 24, 2018 at 12:42:42PM -0700, platerza wrote:
> On my cloned repo I made `srcpkgs/font-sourcecodepro/` a symbolic link to `srcpkgs/font-adobe-source-code-pro` (to match the gnome example) but after pushing it seems like git sees it as the contents of the link and not a symlink itself. Not sure how to do that.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2084#issuecomment-415862921

cd srcpkgs && rm -f font-sourcecodepro && ln -s font-adobe-source-code-pro font-sourcecodepro

**Comment by platerza** [comment]
huh, I'm at a loss for why it's not keeping the link after the push

**Comment by platerza** [comment]
false alarm. was looking at the wrong branch... derp. I'll get to squashing the commits

**Comment by platerza** [comment]
@maxice8 I used `git reset --soft HEAD~4`, `git commit -m "..."`. On the `git push` it won't let me because the branch is now behind. How can I handle this?

**Comment by maxice8** [comment]
On Fri, Aug 24, 2018 at 01:10:47PM -0700, platerza wrote:
> @maxice8 I used `git reset --soft HEAD~4`, `git commit -m "..."`. On the `git push` it won't let me because the branch is now behind. How can I handle this?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2084#issuecomment-415869576

git push -f

**Comment by platerza** [comment]
all done.

**Comment by platerza** [comment]
and squashed the commits again

File: srcpkgs/font-adobe-source-code-pro/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+revision=2
+noarch=yes
+build_style=fetch
+depends="font-util"
+short_desc="Monospaced font family for user interface and coding environments"
+homepage="https://adobe-fonts.github.io/source-code-pro/"
+maintainer="Piero La Terza <platerza@protonmail.com>"
+license="OFL-1.1"
+distfiles="https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Roman.otf https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Italic.otf"
+checksum="af8fdd265f6208816fde44062a27b79ce2a594ded44ea96055a1655b6869992d b2ca3a3c1fe0701ad74aa7c66c37972d07b1237197a816a1a5646c7e42a11353"
```
Comment (maxice8) [inline srcpkgs/font-adobe-source-code-pro/template]: cut distfiles and checksum with a newline

like

```
distfiles="url1
 url2"
checksum="checksum1
 checksum2"
```

File: srcpkgs/font-adobe-source-code-pro/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+revision=2
+noarch=yes
+build_style=fetch
+depends="font-util"
+short_desc="Monospaced font family for user interface and coding environments"
+homepage="https://adobe-fonts.github.io/source-code-pro/"
+maintainer="Piero La Terza <platerza@protonmail.com>"
+license="OFL-1.1"
+distfiles="https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Roman.otf
+	https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Italic.otf"
```
Comment (maxice8) [inline srcpkgs/font-adobe-source-code-pro/template]: prefix the new line with a whitespace like it is done with dependencies, see gtk+3

File: srcpkgs/font-adobe-source-code-pro/template [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'font-adobe-source-code-pro'
+pkgname=font-adobe-source-code-pro
+version=1.010
+revision=2
```
Comment (maxice8) [inline srcpkgs/font-adobe-source-code-pro/template]: revision should be 1

Comment (platerza) [reply srcpkgs/font-adobe-source-code-pro/template]: Updated. By the way, when is the revision supposed to be changed, anyways? I read the description in `Manual.md`:

> `revision`: A number that must be set to 1 when the `source package` is created, or updated to a new `upstream version`. This should only be increased when the generated `binary packages` have been modified.

But the description is not clear to me. Does the package stay at `revision=1` even after an update comes and the package is updated?

**Comment by platerza** [comment]
Updated to use a more stable version. The template was reworked a bit as well.

File: srcpkgs/font-adobe-source-code-pro/template [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+revision=1
+wrksrc="source-code-pro-2.030R-ro-1.050R-it"
+noarch=yes
+depends="font-util"
+font_dirs="/usr/share/fonts/OTF/"
+short_desc="Monospaced font family for user interface and coding environments"
+maintainer="Piero La Terza <platerza@protonmail.com>"
+license="OFL-1.1"
+homepage="https://adobe-fonts.github.io/source-code-pro/"
+distfiles="https://github.com/adobe-fonts/source-code-pro/archive/2.030R-ro/1.050R-it.tar.gz"
```
Comment (maxice8) [inline srcpkgs/font-adobe-source-code-pro/template]: Use ${version} instead of hardcoding

File: srcpkgs/font-adobe-source-code-pro/template [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'font-adobe-source-code-pro'
+pkgname=font-adobe-source-code-pro
+version=2.030R
+revision=1
+wrksrc="source-code-pro-2.030R-ro-1.050R-it"
```
Comment (maxice8) [inline srcpkgs/font-adobe-source-code-pro/template]: use ${version} here as well

**Pull request merged**